Make.inc

```diff
@@ -636,8 +636,8 @@ else
 ISX86:=0
 endif
 
-# If we are running on powerpc64, set certain options automatically
-ifneq (,$(findstring powerpc64,$(ARCH)))
+# If we are running on powerpc64 or ppc64, set certain options automatically
+ifneq (,$(findstring pc64,$(ARCH)))
```

Comment:
use `filter` for this, ref https://github.com/JuliaLang/julia/blob/2e1bcc6f05c7130f79c999cee8adcc8352203819/Makefile#L595